src/main/java/org/jenkinsci/plugins/GithubSecurityRealm.java

```diff
@@ -246,6 +258,13 @@ public void marshal(Object source, HierarchicalStreamWriter writer,
             writer.setValue(realm.getOauthScopes());
             writer.endNode();
 
+            writer.startNode("redirectUri");
+            String redirectUriValue = DEFAULT_REDIRECT_URI;
+            if (null != realm.getRedirectUri()) {
+            		redirectUriValue = realm.getRedirectUri();
```

Comment:
Mixing tabs and spaces.  There are a couple of places in your change where this happens so I'll just comment this once.

Reply:
Expanded the tabs.